zephyr/module.yml

```diff
@@ -10,14 +10,4 @@ build:
   # Zephyr build system. The `.` is the root of this repository.
   cmake: .
   settings:
-    # Additional roots for boards and DTS files. Zephyr will use the
-    # `<board_root>/boards` for additional boards. The `.` is the root of this
-    # repository.
-    board_root: .
-    # Zephyr will use the `<dts_root>/dts` for additional dts files and
-    # `<dts_root>/dts/bindings` for additional dts binding files. The `.` is
-    # the root of this repository.
-    dts_root: .
-    # Zephyr will use the `<soc_root>/soc` for additional socs. The `.` is the
-    # root of this repository.
```

Comment:
I'd consider keeping these, so that it remains possible to have boards in this tree in the future. Downstream (to us) users might also want to populate a fork of this tree with their own board definitions possibly (though, a better solution might be to have a separate manifest tree in that case that just points at this one)